src/main/java/nl/ordina/jobcrawler/controller/VacancyScraper.java

```diff
@@ -0,0 +1,32 @@
+package nl.ordina.jobcrawler.controller;
+
+import nl.ordina.jobcrawler.model.Vacancy;
+import nl.ordina.jobcrawler.model.VacancyURLs;
+import org.jsoup.nodes.Document;
+
+import java.io.IOException;
+import java.util.List;
+
+// All vacancy scrapers have the following functions in common
+
+abstract class VacancyScraper {
```

Comment:
Zou dit niet beter een interface kunnen zijn?

Reply:
Dit was een interface, maar is naar mijn aanvraag veranderd naar een abstract class omdat er waarschijnlijk ook overeenkomstige variabelen zullen worden geïntroduceerd tussen scrapers

Reply:
Ik denk dat de manier waarop ik dit nu heb gedaan dit een abstract class moet zijn. Enkele methodes met body en enkele abstract. Zie https://github.com/mostafa-221/jobcrawler-backend/commit/d07d302f69fdef151383e4a3c1b6b49a752f0064